### DISCUSSION TITLE
Regarding cbor dependencies for server functions.

### DISCUSSION THREAD

**Discussion by afiqzx** [leptos-rs/leptos#705] [General]
I can't help but notice the current server functions are dependending on `serde_urlencoded` or the [`cbor` ](https://github.com/BurntSushi/rust-cbor) crate. After further checking cbor is actually unmaintained, as written in the first line of its README.

Andrew Gallant has linked to [`serde_cbor`](https://github.com/pyfisch/cbor) which is also unmaintained, but the author linked [`ciborium`](https://github.com/enarx/ciborium) and [`minicbor`](https://gitlab.com/twittner/minicbor) which looks maintained. Both looks good and I am leaning towards `ciborium` as it depends on `serde` (we already has `serde_urlencoded` so it should be good).

Personally I have no problems using the current cbor dependencies but it would be nice to have a properly maintained dependencies moving forward.

If you have no problems in changing current cbor dependency, I would like to create an issue on work on this.

**Comment by gbj** [comment]
The server functions currently depend on `ciborium`. I don't think we depend on the `cbor` crate anywhere, as distinct from referring to it as the source of the CBOR encoding in docs, but I may be wrong.

**Comment by benwis** [comment]
Yep, I might have added cbor for some early testing, but we are only using ciborium now. If you see that as a dep somewhere, let us know.
